README.md

```diff
@@ -6,4 +6,23 @@ This is the third project of the Main Ruby curriculum at Microverse. The goal is
 ## Andres F. Moya https://github.com/AndresFMoya/
 
 # 1. Usage Instructions
-Clone the repo and run the app: ruby 'bin/main.rb'. The game accepts only integers from 1 to 9. The player who succeeds in placing three of their marks in a row wins the game, or it's draw.
\ No newline at end of file
+### clone the repo: git clone https://github.com/rOluochKe/tictactoe.git
+### change directory: cd tictactoe
+### run the program: ruby 'bin/main.rb'
+### enjoy play the game
+
+# 2. Rules 
+### The game accepts only integers from 1 to 9. The player who succeeds in placing three of their marks in a row wins the game, or it's draw.
```

Comment:
Try to be more descriptive with the rules. Who starts first, how does he insert the position in which he wants to put his mark on the board, etc.

Reply:
resolved